components/net/resource_thread.rs

```diff
@@ -436,15 +464,16 @@ impl ResourceManager {
             "chrome" => from_factory(chrome_loader::factory),
             "file" => from_factory(file_loader::factory),
             "http" | "https" | "view-source" => {
+                let resource_grp = self.get_resource_group(load_data.pipeline_id);
```

Comment:
Why would the scheme be a file:// url? The tests are loaded from a local HTTP server.